cf/field.py

```diff
@@ -14458,163 +14458,197 @@ def where(
+                  values.
+
+                *Parameter example:*
+                  ``f.where(cf.lt(0), x=-f.data, y=cf.masked)`` will
+                  change the sign of all negative data values, and set
+                  all other data values to missing data.
+
+                If *x* or *y* is a `Field` then it is first
+                transformed so that its data is broadcastable to the
+                data being assigned to. This is done by using the
+                metadata constructs of the two field constructs to
+                create a mapping of physically identical dimensions
+                between the fields, and then manipulating the
+                dimensions of other field construct's data to ensure
+                that they are broadcastable. If either of the field
+                constructs does not have sufficient metadata to create
+                such a mapping then an exception will be raised. In
+                this case, any manipulation of the dimensions must be
+                done manually, and the `Data` instance of *x* or *y*
+                (rather than the field construct itself) may be used
+                for the condition.
 
                 *Parameter example:*
-                  If field construct ``f`` has shape ``(5, 3)`` and ``g =
-                  f.transpose() * 10`` then ``f.where(cf.lt(0), x=g)``
-                  will set all data values that are less than zero to the
-                  equivalent elements of field construct ``g``, provided
-                  there are sufficient metadata for the data dimensions to
-                  be mapped. However, ``f.where(cf.lt(0), x=g.data)`` will
-                  always fail in this example, because the shape of the
-                  condition is ``(3, 5)``, which does not broadcast to the
-                  shape of the ``f``.
+                  If field construct ``f`` has shape ``(5, 3)`` and
+                  ``g = f.transpose() * 10`` then ``f.where(cf.lt(0),
+                  x=g)`` will set all data values that are less than
+                  zero to the equivalent elements of field construct
+                  ``g``, provided there are sufficient metadata for
+                  the data dimensions to be mapped. However,
+                  ``f.where(cf.lt(0), x=g.data)`` will always fail in
+                  this example, because the shape of the condition is
+                  ``(3, 5)``, which does not broadcast to the shape of
+                  the ``f``.
 
             construct: `str`, optional
-                Define the condition by applying the *construct* parameter
-                to the given metadata construct's data, rather then the
-                data of the field construct. Must be
+                Define the condition by applying the *construct*
+                parameter to the given metadata construct's data,
+                rather then the data of the field construct. Must be
```

Comment:
A previous typo but we might as well correct it now:
```suggestion
                rather than the data of the field construct. Must be
```